sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -390,6 +390,44 @@ export interface DirectoryProperties extends FileAndDirectorySetPropertiesCommon
     abortSignal?: AbortSignalLike;
 }
 
+// @public
+export interface DirectoryRenameHeaders {
+    date?: Date;
+    etag?: string;
+    fileAttributes?: string;
+    fileChangeTime?: Date;
+    fileCreationTime?: Date;
+    fileId?: string;
+    fileLastWriteTime?: Date;
+    fileParentId?: string;
+    filePermissionKey?: string;
+    isServerEncrypted?: boolean;
+    lastModified?: Date;
+    requestId?: string;
+    version?: string;
+}
+
+// @public
+export interface DirectoryRenameOptions extends CommonOptions {
+    abortSignal?: AbortSignalLike;
+    copyFileSmbInfo?: CopyFileSmbInfo;
```

Comment:
>CopyFileSmbInfo [](http://example.com/codeflow?start=22&length=15)

Is this aligning to .Net's API? Understand the elements in CopyFileSmbInfo is same to rename from functional aspect. While worth we extend the interface as RenameFileSmbInfo?

Reply:
.Net's copy and rename use the same structure which is named: FileSmbProperties.  We don't want to break the old interfaces, so didn't change its name.

Reply:
CopyFileSmbInfo is the one used in Swagger spec. We'll keep it for now.

Reply:
May defer to API review, would better create a PBI to trace.